### ISSUE TITLE
How to change 'poly_lr' to other lr_scheduler in pytorch

### ISSUE THREAD

**Issue by yan4821567** [MIC-DKFZ/nnUNet#569]
Hi Fabian,

Thanks for your work. I heard that CosineAnnealingLR is a useful trick to improve accuracy. I want to add this learning rate to nnUnet.
And I found 'poly_lr' is used in :
nnunet.training.network_training.nnUNetTrainerV2.py #line407
`self.optimizer.param_groups[0]['lr'] = poly_lr(ep, self.max_num_epochs, self.initial_lr, 0.9)`
It seems that learning rate in nnUNet is encapsulated in the optimizer

But other learning rate like CosineAnnealingLR is defined as : 
`scheduler = optim.lr_scheduler.CosineAnnealingLR(optimizer, T_max=5,eta_min=4e-08)`
`for epoch in range(100):`
`scheduler.step()`
`train(…)`
`validate(…)`
 
I don't know how to change 'poly_lr' to others in nnUnet. Can you give some advice?

**Comment by FabianIsensee** [comment]
I have implemented that already and it has not improved results relative to the current schedule :-)
But maybe_update_lr is already the correct place to look :-) 
Best,
Fabian

**Comment by yan4821567** [comment]
> I have implemented that already and it has not improved results relative to the current schedule :-)
> But maybe_update_lr is already the correct place to look :-)
> Best,
> Fabian

Thank you!
